s3-import-commons/src/main/java/no/unit/nva/publication/s3imports/FilenameEventEmitter.java

```diff
@@ -163,7 +163,7 @@ private void logWarningForNotEmittedFilenames(List<PutEventsResult> failedReques
                                                  .map(this::newImportRequestForSingleFile)
                                                  .collect(Collectors.toList());
         eventEmitter.addEvents(filenameEvents);
-        return eventEmitter.emitEvents(10, WAIT_TIME_IN_MILLIS_FOR_EMITTING_BATCHES_OF_FILENAMES);
+        return eventEmitter.emitEvents(10);
```

Comment:
What does '10' describe? Could it be substituted with an constant, NUMBER_OF_EVENTS_TO_EMIT or EVENT_BATCH_SIZE?